demos/classification_demo/python/classification_demo.py

```diff
@@ -39,71 +39,87 @@
-                         help='Optional. List of monitors to show initially.')
+        help='enable reading the input in a loop')
+
+    io_args.add_argument('-o', '--output', required=False, metavar="<OUTPUT>",
+        help='name of the output file(s) to save')
+
+    io_args.add_argument('-limit', '--output_limit', required=False, default=1000, type=int, metavar="<NUMBER>",
+        help='number of frames to store in output. If 0 is set, all frames are stored. Default value is 1000')
+
+    io_args.add_argument('--no_show', action='store_true',
+        help="don't show output")
+
+    io_args.add_argument('--output_resolution', default=None, type=resolution, metavar="<STRING>",
+        help='specify the maximum output window resolution in (width x height) format. Example: 1280x720'
+            'Default value is input frame size')
+
+    io_args.add_argument('-u', '--utilization_monitors', default='', type=str, metavar="<DEVICE>",
+        help='list of monitors to show initially')
 
     input_transform_args = parser.add_argument_group('Input transform options')
+
     input_transform_args.add_argument('--reverse_input_channels', default=False, action='store_true',
-                                      help='Optional. Switch the input channels order from '
-                                           'BGR to RGB.')
-    input_transform_args.add_argument('--mean_values', default=None, type=float, nargs=3,
-                                      help='Optional. Normalize input by subtracting the mean '
-                                           'values per channel. Example: 255.0 255.0 255.0')
-    input_transform_args.add_argument('--scale_values', default=None, type=float, nargs=3,
-                                      help='Optional. Divide input by scale values per channel. '
-                                           'Division is applied after mean values subtraction. '
-                                           'Example: 255.0 255.0 255.0')
+        help='switch the input channels order from BGR to RGB')
+
+    input_transform_args.add_argument('--mean_values', default=None, type=float, nargs=3, metavar="<STRING>",
+        help='normalize input by subtracting the mean values per channel. Example: 255.0 255.0 255.0')
+
+    input_transform_args.add_argument('--scale_values', default=None, type=float, nargs=3, metavar="<STRING>",
+        help='divide input by scale values per channel(division is applied after mean values subtraction). '
+            'Example: 255.0 255.0 255.0')
 
     debug_args = parser.add_argument_group('Debug options')
-    debug_args.add_argument('-r', '--raw_output_message', help='Optional. Output inference results raw values showing.',
-                            default=False, action='store_true')
+
+    debug_args.add_argument('-r', '--raw_output_message', default=False, action='store_true',
+        help='output inference results raw values showing')
+
     return parser
 
 
```

Comment:
Replace `sys.exit(main() or 0)` with just `main()` at the bottom

Reply:
Done